front/.eslintrc

```diff
@@ -46,22 +46,6 @@
     "@typescript-eslint/no-unnecessary-type-assertion": "error",
     "@typescript-eslint/no-unsafe-return": "error",
     "@typescript-eslint/no-unsafe-call": "error",
-
-    "@typescript-eslint/ban-types": [
```

Comment:
`ban-types` has been superseded by `no-restricted-types` it seems?

https://typescript-eslint.io/rules/no-restricted-types

Reply:
Yes exactly. I started to replace it but I noticed we don't use FC anymore in the app. 
Do you want to keep this rule in case someone was tempted to use one ?

Reply:
We added the rule back when we removed all FC uses to make sure we don't re-introduce them by mistake IIRC.

Reply:
Put the rule back